west.yml

```diff
@@ -249,7 +249,7 @@ manifest:
       groups:
         - hal
     - name: hal_xtensa
-      revision: 08325d6fb7190a105f5382d35e64ed2812c57cf4
+      revision: pull/26/head
       path: modules/hal/xtensa
```

Comment:
Please update this with the hash of the merged commit or last existing commit in hal_xtensa.